uw_sws/models.py

```diff
@@ -809,15 +809,25 @@ class Registration(models.Model):
     is_credit = models.NullBooleanField()
     is_auditor = models.NullBooleanField()
     is_independent_start = models.NullBooleanField()
-    start_date = models.DateField(blank=True)
-    end_date = models.DateField(blank=True)
-    request_date = models.DateField(blank=True)
+    start_date = models.DateField(blank=True, null=True)
+    end_date = models.DateField(blank=True, null=True)
+    request_date = models.DateField(blank=True, null=True)
```

Comment:
Why did we add null=True here?

Reply:
Not necessary.
    if len(reg_data["StartDate"]) > 0:
        registration.start_date = parse(reg_data["StartDate"])
    if len(reg_data["EndDate"]) > 0:
        registration.end_date = parse(reg_data["EndDate"])
    if len(reg_data["RequestDate"]) > 0:
        registration.request_date = parse(reg_data["RequestDate"])
We may want to set the default value.